doc/nrf/images/zigbee_signal_handler_05_formation.uml

```diff
@@ -0,0 +1,13 @@
+@startuml
+
+:ZB_BDB_SIGNAL_FORMATION;
+
+if (status == RET_OK) then (no)
+    :bdb_start_top_level_commissioning(ZB_BDB_NETWORK_FORMATION);
+else (yes)
+    :start_network_rejoin();
```

Comment:
It's _bdb_start_top_level_commissioning(ZB_BDB_NETWORK_STEERING)_

Reply:
Are you sure about that?
```
		LOG_INF("Device started for the first time");
		if (status == RET_OK) {
			if (role != ZB_NWK_DEVICE_TYPE_COORDINATOR) {
				LOG_INF("Start network steering");
				start_network_rejoin();
			} else {
				LOG_INF("Start network formation");
				comm_status = bdb_start_top_level_commissioning(
					ZB_BDB_NETWORK_FORMATION);
			}
		} else {
			LOG_ERR("Failed to initialize Zigbee stack (status: %d)",
				status);
		}
		break;
```

Reply:
Agree. The current flow should have the network steering there:
```
if (status == RET_OK) {
	comm_status = bdb_start_top_level_commissioning(ZB_BDB_NETWORK_STEERING);
} else {
	ret_code = ZB_SCHEDULE_APP_ALARM(bdb_start_top_level_commissioning, ZB_BDB_NETWORK_FORMATION, ZB_TIME_ONE_SECOND);
}
```

Reply:
Still not fixed.

Reply:
Fixed